examples/deck/example.js

```diff
@@ -119,21 +168,23 @@ export default class Example extends Component<
           }
         });
       },
-      onStopDraggingPoint: ({ coordinateIndexes }) => {
-        // eslint-disable-next-line no-console, no-undef
-        console.log(`Stop dragging point`, coordinateIndexes);
+      onStopDraggingPoint: ({ featureIndex, coordinateIndexes }) => {
+        console.log(`Stop dragging point`, featureIndex, coordinateIndexes); // eslint-disable-line
       },
 
-      // Can specify GeoJsonLayer props
-      getFillColor: () => [0x00, 0x20, 0x70, 0x30],
-      getLineColor: () => [0x00, 0x20, 0x70, 0xc0],
-      getLineWidth: () => 30,
+      // Specify the same GeoJsonLayer props
       lineWidthMinPixels: 2,
-      lineWidthMaxPixels: 10,
 
-      // As well as point layer props
-      getPointColor: () => [0x00, 0x20, 0x70, 0xff],
-      pointHighlightColor: [0xff, 0xff, 0xff, 0xff]
+      // Accessors receive an isSelected argument
+      getFillColor: (feature, isSelected) => {
+        return isSelected ? [0x20, 0x40, 0x90, 0xc0] : [0x20, 0x20, 0x20, 0x30];
```

Comment:
Don't you need some `updateTrigger` to make sure that a change in `isSelected` regenerates the layer's color attributes?

Reply:
that's in `EditableGeoJsonLayer` itself

Reply:
Oh I see, isSelected is a parameter. Makes sense then.